js_modules/dagster-ui/packages/ui-components/src/components/ifPlural.ts

```diff
@@ -0,0 +1,3 @@
+export function ifPlural(count: number | undefined | null, string: string) {
+  return count === 1 ? '' : string;
+}
```

Comment:
This is okay, though it might not really help if the verb disagrees. I tend to just go with writing out the entire string, since I find it easier to comprehend the copy if I don't have to mentally add an `s` (or whatever the declension is) to the noun:

```
count === 1 ? '1 run requested' : `${count} runs requested`
```

Doesn't matter a ton for now anyway, as long as we're providing appropriate strings, until the point in the future when we decide to internationalize the app.

Reply:
Ah yeah I guess I should make you pass in both versions of the string instead